docs/src/service-examples.md

```
@@ -80,3 +80,34 @@ To run unit tests:
 ```console
 $ docker run --rm -v $(pwd):/app jade test
 ```
+
+### as-lan
+
+The as-lan docker image ships with Node.js, `wasm-pvm`, and the AssemblyScript toolchain pre-installed. Pull it:
+
+```console
+$ docker pull ghcr.io/tomusdrw/jammin-as-lan:0.0.4
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)